### PR TITLE
fix checking wrong variable naming should be case insensitive

### LINKED ISSUE #1275: `SOME = 1` is allowed
Turns out our check is case sensitive. 

It should should raise `WPS110` as `some = 1` does.

### PR DESCRIPTION
**Pull request by ammarnajjar**
Fix checking wrong variable naming should be case insensitive.

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues

- Closes #1275

### CODE REVIEW

File: tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py [hunk 1]
```diff
@@ -1,11 +1,41 @@
...
+    """Ensures that wrong names are not allowed, case insensitive."""
+    tree = parse_ast_tree(mode(naming_template.format(wrong_name)))
+
+    visitor = WrongNameVisitor(default_options, tree=tree)
+    visitor.run()
+
+    assert_errors(
+        visitor,
+        [WrongVariableNameViolation],
+        ignored_types=UpperCaseAttributeViolation,
```
Comment (sobolevn) [inline tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py]: `ignored_types=(UpperCaseAttributeViolation,),`

File: tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py [hunk 2]
```diff
@@ -1,11 +1,41 @@
...
 
 from wemake_python_styleguide.violations.naming import (
+    UpperCaseAttributeViolation,
     WrongVariableNameViolation,
 )
 from wemake_python_styleguide.visitors.ast.naming import WrongNameVisitor
 
 
+@pytest.mark.parametrize('wrong_name', [
+    'SOME',
```
Comment (sobolevn) [inline tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py]: It looks like we are missing a template in our cases here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_naming/conftest.py

```python
class_name = 'class {0}: ...'
```

Can you please also add it?

File: tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py (hunk 1 above)
Comment (ammarnajjar) [reply tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py]: d80c65e

File: tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py (hunk 2 above)
Comment (ammarnajjar) [reply tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py]: sure => a90b5ae

**Comment by sobolevn** [comment]
Thanks a lot! Awesome job! 🌮

**Pull request merged**